network/src/protocols/rpc/mod.rs

```diff
@@ -475,7 +476,7 @@ async fn handle_outbound_rpc_inner(
     let prototol_id_descriptor = format!("{:?}", protocol);
     // Start timer to collect RPC latency.
     let timer = counters::LIBRA_NETWORK_RPC_LATENCY
-        .with_label_values(&["request", &prototol_id_descriptor])
+        .with_label_values(&["request", &prototol_id_descriptor, &peer_id_str])
```

Comment:
nit: use `peer_id.short_str()` here for peer id label

in fact, you might as well set `peer_id_str` to `peer_id.short_str()` and re-use it for the trace logs

Reply:
This is actually one thing I have some questions about. The `short_str` takes the 4 characters of the account address, so would there be the chance of two peer IDs having the same `short_str` going into the same histogram partition?